drivers/gpio/gpio_renesas_ra_ioport.c

```diff
@@ -9,21 +9,56 @@
 #include <zephyr/drivers/gpio.h>
 #include <zephyr/drivers/pinctrl.h>
 #include <zephyr/dt-bindings/gpio/renesas-ra-gpio-ioport.h>
+#include <zephyr/drivers/misc/renesas_ra_external_interrupt/renesas_ra_external_interrupt.h>
 #include <zephyr/drivers/gpio/gpio_utils.h>
-#include <zephyr/irq.h>
 #include <soc.h>
 
+struct gpio_ra_irq_info {
```

Comment:
I think this implementation is somewhat based on mine, but it wasn't sophisticated enough, so the way this data is held could be made a bit more efficient.

Reply:
Hi @soburi , could you please give me more about this comment? Which actions should be done for it?

Reply:
I found my previously considered improving source and looked at it again, but it wasn't all that efficient, so this is fine.
Sorry for stopping.